generation/um/wincon/generate.rsp

```diff
@@ -0,0 +1,16 @@
+@../../settings.rsp
+@../../remap.rsp
+--file
+um-wincon.h
+--output
+../../../sources/Interop/Windows/um/wincon
+--test-output
+../../../tests/Interop/Windows/um/wincon
+--traverse
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.19041.0/um/consoleapi.h
```

Comment:
For TerraFX, we should only be traversing a single file from each `generate.rsp`

So separate `generate.rsp` are needed for `consoleapi.h`, etc

Reply:
That is slightly unfortunate, since I was hoping that I could just go through `wincon.h` once. I'll split it into multiple directories